Automattic/wp-calypso
# Theme Showcase: Revert new theme preview modal

## Proposed Changes

This PR removes code that implements the new theme preview modal via the feature flag `themes/showcase-i4/details-and-preview`. Since we are bringing style selection in the theme detail page instead, we'll move the Tracks events there after the work in the theme detail page is done.

## Testing Instructions



* Head to the logged-in Theme Showcase `/themes/${site_slug}`. If using calypso.live, the flag `themes/showcase-i4/details-and-preview` is required.
* Ensure that the live demo button opens up the old theme preview modal, not the new one.

## Pre-merge Checklist
